docs/guides/12-add-widgets-advanced.md

```diff
@@ -0,0 +1,262 @@
+## Add widgets advanced
+
+We have seen how to add legends to our map by using `getLegendsData` method, and we have also learned how to display widgets using histogram expressions in the [previous guide](/developers/carto-vl/guides/add-widgets/). This is an advanced guide to learn how to obtain and display the information in widgets by using `viewportHistogram` and `globalHistogram` expressions.
+
+### Overview
+
+We will learn how to create bar charts for **categorical** data and histograms for **numeric** data. For this purpose, the following examples make use of the same source, the [**Vancouver Trees**](https://team.carto.com/u/cartovl/tables/cartovl.vancouver_trees/public/map) dataset.
+
+Bar charts and histograms have been built with [Chart.js](https://www.chartjs.org), a very simple external library. In the previous guide we used [Airship](https://carto.com/airship/) to build a histogram widget. In this case, we want to demonstrate you can use the visualization library of your choice with CARTO VL, and therefore, we have to explain how the histogram expressions work a bit deeply.
```

Comment:
```suggestion
Bar charts and histograms have been built with [Chart.js](https://www.chartjs.org), a very simple external library. In the previous guide we used [Airship](https://carto.com/airship/), which is the natural choice to build a histogram widget, but in this case we want to demonstrate how the histogram expressions work a bit deeply.
```

I would stress a bit the idea of VL <--> Airship here. That will be improved a lot with the Airship v2 VL-Bridge

Reply:
I agree, but I think this is not the place to do so, or maybe not at this time. Maybe we can modify this when there's a VL- Airship bridges guide, what do yo think? Explaining something that is not yet available adds a bit of noise, in my opinion.